frontend/app/page.tsx

```
@@ -0,0 +1,31 @@
+        <p className="text-2xl mb-8">Architecture Blueprint - Coming Soon</p>
+        <div className="grid grid-cols-1 md:grid-cols-3 gap-6 mt-12">
+          <div className="bg-white/10 backdrop-blur-lg rounded-lg p-6">
+            <h2 className="text-xl font-bold mb-2">Auth Service</h2>
+            <p className="text-sm">User authentication & management</p>
+          </div>
+          <div className="bg-white/10 backdrop-blur-lg rounded-lg p-6">
+            <h2 className="text-xl font-bold mb-2">Chat Service</h2>
+            <p className="text-sm">Real-time messaging</p>
+            <a 
+              href="http://localhost:3001" 
+              target="_blank"
```

Biome (2.3.13)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)